src/scipp/ndimage/__init__.py

```diff
@@ -0,0 +1,108 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+# @author Simon Heybrock
+"""Sub-package for multidimensional image processing.
+
+This subpackage provides wrappers for a subset of functions from
+:py:mod:`scipy.ndimage`.
+"""
+from functools import wraps
+from typing import Callable, Union
+
+from ..core import Variable, DataArray
+from ..core import CoordError, DimensionError
+from ..core import empty_like, islinspace, ones
+
+
+def ndfilter(func: Callable) -> Callable:
+
+    @wraps(func)
+    def function(x: Union[Variable, DataArray], **kwargs) -> Union[Variable, DataArray]:
+        if 'output' in kwargs:
+            raise ValueError("The 'output' argument is not supported")
```

Comment:
I think this should be `TypeError`. This is what you usually get for specifying arguments that don't exist.